requirements.txt

```diff
@@ -1 +1,2 @@
 prompt-toolkit==3.0.14
+typing_extensions; python_version < '3.8'
```

Comment:
Does this also need to be added to `install_requires` in `setup.cfg`? (May not have been caught by CI because the `verify_wheel` CI step only runs on 3.9.1)

Reply:
Yes, it should be added to `setup.cfg`. Good catch!